base_layer/wallet/src/key_manager_service/handle.rs

```diff
@@ -73,6 +73,19 @@ where TBackend: KeyManagerBackend + 'static
         (*self.key_manager_inner).write().await.apply_encryption(cipher)
     }
 
+    async fn create_key_pair(&self, key_seed: String) -> Result<(PrivateKey, PublicKey), KeyManagerServiceError> {
+        (*self.key_manager_inner)
```

Comment:
nit: I know this is copied from other parts of the code, but there is no need to explicitly dereference an Arc when using the `.` operator
```suggestion
        self.key_manager_inner
```

Reply:
This seems like something clippy should be able to catch

Reply:
Definitely, I see some other parts of the code in which this is also the case. Should these be addressed @sdbondi ? Thanks for the catch :)

@delta1, maybe because the types of `self.key_manager_inner` and `(*self.key_manager_inner)` are distinct ?

Reply:
yeah I would vote for clippy catching these, but until then, not important at all to change this (nit) feel free to leave as is.